example/contract-schema.ts

```diff
@@ -3,12 +3,12 @@ import { InMemorySigner } from '@taquito/signer';
 
 
 async function example() {
-    const provider = 'https://api.tez.ie/rpc/florencenet';
+    const provider = 'https://api.tez.ie/rpc/granadanet';
     const signer: any = new InMemorySigner('edskRtmEwZxRzwd1obV9pJzAoLoxXFWTSHbgqpDBRHx1Ktzo5yVuJ37e2R4nzjLnNbxFU4UiBU1iHzAy52pK5YBRpaFwLbByca');
     const tezos = new TezosToolkit(provider);
     tezos.setSignerProvider( signer );
     try {
-        const contract = await tezos.contract.at('KT1SwtWpUE1TpBF8c4T1c2NsaDGgjKv3Xwy7');
+        const contract = await tezos.contract.at('KT1D96WbVea1S4ZioZ6dtvwoKa1T1nXz4qhz');
```

Comment:
The example throws the following error:
```
TezosOperationError {
  errors: [
    {
      kind: 'temporary',
      id: 'proto.010-PtGRANAD.michelson_v1.runtime_error',
      contract_handle: 'KT1D96WbVea1S4ZioZ6dtvwoKa1T1nXz4qhz',
      contract_code: [Array]
    },
    {
      kind: 'temporary',
      id: 'proto.010-PtGRANAD.michelson_v1.script_rejected',
      location: 1903,
      with: [Object]
    }
  ],
  name: 'TezosOperationError',
  id: 'proto.010-PtGRANAD.michelson_v1.script_rejected',
  kind: 'temporary',
  message: 'WrongCondition: sp.sender == self.data.administrator'
}
```

Reply:
I am not able to figure out how to resolve this one. 
I would like to eave this one as Florence for now and get the other changes merged.
I'll make a new PR to update this example to Grenada.

Reply:
https://github.com/ecadlabs/taquito/issues/1013